### PR TITLE
updated code for wheel base

### PR DESCRIPTION
**Pull request by Prashanttiwari2190**

### CODE REVIEW

**Comment by Prashanttiwari2190** [comment]
@abhinavatai pleases review the pR

File: src/SFM.VehicleBuilder.Application/Queries/UXGetModeByStylesQuery/UXGetModeByStylesQueryHandler.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            var modelConfigration = new ModelConfigration();
+            var response = await chromeDataService.GetModelConfigurationByStyleIds(request.StyleIds);
+            if (response != null)
+            {
+               var responseWheelbase = response.configuration.technicalSpecifications.Where(s => s.titleId == 301);
+               var responsecolors = response.configuration.equipment.Where(s => s.headerName == "PRIMARY PAINT");
+
+               if (responseWheelbase.Any())
+                {
+                foreach (var wheel in responseWheelbase)
```
Comment (abhinavatai) [inline src/SFM.VehicleBuilder.Application/Queries/UXGetModeByStylesQuery/UXGetModeByStylesQueryHandler.cs:39]: Can't this foreach be optimized using linq?

File: src/SFM.VehicleBuilder.Application/Queries/UXGetModeByStylesQuery/UXGetModeByStylesQueryHandler.cs [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+using ChromeData;
+using MediatR;
+using SFM.VehicleBuilder.Data.Services.ChromeData;
+using SFM.VehicleBuilder.Domain.Models;
+using SFM.VehicleBuilder.Domain.Models.ChromeData;
+using SFM.VehicleBuilder.Domain.Models.SearchStyle;
+
+namespace SFM.VehicleBuilder.Application.Queries.UXGetModeByStylesQuery
+{
+    public class UXGetModeByStylesQueryHandler : IRequestHandler<UXGetModeByStylesQuery, ModelConfigration>
```
Comment (abhinavatai) [inline src/SFM.VehicleBuilder.Application/Queries/UXGetModeByStylesQuery/UXGetModeByStylesQueryHandler.cs:17]: class name should be GetModelConfigByStyles @Prashanttiwari2190

File: src/SFM.VehicleBuilder.Data/Connected Services/ChromeData/Reference.cs [hunk 3]
```diff
@@ -12181,7 +12181,7 @@ public partial class getModelConfigurationByStyleIdsRequest
         
         [System.ServiceModel.MessageBodyMemberAttribute(Namespace="urn:configcompare4h.kp.chrome.com", Order=0)]
         public ChromeData.ModelConfigurationByStyleIdsRequest ModelConfigurationByStyleIdsRequest;
-
```
Comment (abhinavatai) [inline src/SFM.VehicleBuilder.Data/Connected Services/ChromeData/Reference.cs:12184]: It's just a blank space. But please never commit this file.

File: src/SFM.VehicleBuilder.Data/Services/ChromeData/ChromeDataService.cs [hunk 4]
```diff
@@ -1,7 +1,9 @@
 ﻿using ChromeData;
+using DocumentFormat.OpenXml.Office2010.Excel;
```
Comment (abhinavatai) [inline src/SFM.VehicleBuilder.Data/Services/ChromeData/ChromeDataService.cs:2]: remove unused dependency. @Prashanttiwari2190

File: src/SFM.VehicleBuilder.Domain/Models/SearchStyle/GenericColors.cs [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+{
+    /// <summary>
+    ///   A chrome data vehicle GenericColors.
+    /// </summary>
+    public class GenericColors
+    {
+        /// <summary>
+        ///   Gets or sets a value indicating whether gets or sets a chrome data Name.
+        /// </summary>
+        public string[] Name { get; set; }
```
Comment (abhinavatai) [inline src/SFM.VehicleBuilder.Domain/Models/SearchStyle/GenericColors.cs:18]: should be string only not string[]?

File: src/SFM.VehicleBuilder.Domain/Models/SearchStyle/WheelBase.cs [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+{
+    /// <summary>
+    ///   A chrome data vehicle CargoItem.
+    /// </summary>
+    public class WheelBase
+    {
+        /// <summary>
+        ///   Gets or sets a value indicating whether a chrome data Value.
+        /// </summary>
+        public IEnumerable<string> Value { get; set; }
```
Comment (abhinavatai) [inline src/SFM.VehicleBuilder.Domain/Models/SearchStyle/WheelBase.cs:18]: should be string instead IE<string> ?

File: src/SFM.VehicleBuilder.Web/Controllers/V1.0/UxVehicleController.cs [hunk 7]
```diff
@@ -176,5 +177,22 @@ public async Task<ActionResult<IEnumerable<Styles>>> StyleSearch([FromBody] Styl
 
             return await this.Execute(logger, () => mediator.Send(query, CancellationToken.None));
         }
+
+        /// <summary>
+        ///   Gets the StyleSearch list.
+        /// </summary>
+        /// <param name="styleids"> division information.</param>
+        /// <returns>Returns a <see cref="int"/> containg the ModelConfigration of the styles.</returns>
+        [HttpPost("model-style-search")]
```
Comment (abhinavatai) [inline src/SFM.VehicleBuilder.Web/Controllers/V1.0/UxVehicleController.cs:186]: route should be "styles-config"

Comment (Prashanttiwari2190) [reply src/SFM.VehicleBuilder.Web/Controllers/V1.0/UxVehicleController.cs:186]: ok

File: src/SFM.VehicleBuilder.Data/Services/ChromeData/ChromeDataService.cs (hunk 4 above)
Comment (Prashanttiwari2190) [reply src/SFM.VehicleBuilder.Data/Services/ChromeData/ChromeDataService.cs:2]: i have not added.

File: src/SFM.VehicleBuilder.Application/Queries/UXGetModeByStylesQuery/UXGetModeByStylesQueryHandler.cs (hunk 2 above)
Comment (Prashanttiwari2190) [reply src/SFM.VehicleBuilder.Application/Queries/UXGetModeByStylesQuery/UXGetModeByStylesQueryHandler.cs:17]: ok

**Pull request merged**